app/clients/sms/firetext.py

```diff
@@ -6,26 +6,34 @@
 )
 from flask import current_app
 from requests import request, RequestException, HTTPError
+from app.clients import ClientResponse, STATISTICS_DELIVERED, STATISTICS_FAILURE
 
 logger = logging.getLogger(__name__)
 
-firetext_response_status = {
-    '0': {
-        "firetext_message": 'delivered',
-        "success": True,
-        "notify_status": 'delivered'
-    },
-    '1': {
-        "firetext_message": 'declined',
-        "success": False,
-        "notify_status": 'failed'
-    },
-    '2': {
-        "firetext_message": 'Undelivered (Pending with Network)',
-        "success": False,
-        "notify_status": 'sent'
-    }
-}
+
+class FiretextResponses(ClientResponse):
```

Comment:
Firetext verison. Note here the "pending" state doesn't update the stats table (None state)